src/components/views/messages/EditHistoryMessage.tsx

```diff
@@ -141,9 +141,13 @@ export default class EditHistoryMessage extends React.PureComponent<IProps, ISta
             );
         }
 
+        const classes = classNames("mx_MessageActionBar", {
+            "mx_MessageActionBar--noButton": !SettingsStore.getValue("developerMode") && !redactButton,
+        });
+
```

Comment:
The element should be removed from the DOM rather than hidden with css.
Something like:
```
if (!redactButton && !viewSourceButton) return null;
```

Reply:
That totally makes sense, thanks!